feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/ConversationViewModel.kt

```diff
@@ -380,6 +382,7 @@ internal open class ConversationViewModel(
             )
                 .bind { if (it) null else MetisModificationFailure.DELETE_POST }
                 .or(MetisModificationFailure.DELETE_POST)
+                .also { if (it != MetisModificationFailure.DELETE_POST && post is IStandalonePost && onCloseThread != null) onCloseThread?.invoke() }
```

Comment:
I think `&& onCloseThread != null` is not needed here due to the  null coalescing operator in the invoke call

Reply:
Yes, you're right!